src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
calendar should have modifiers (for amount of days and start day). Changing this modifiers we should see changed calendar

Reply:
Oh, sure. Sorry. https://TetianaBukhenko.github.io/layout_calendar/

Reply:
May I clarify (I probably got the task wrong), I should add modifiers to the main container "calendar", right? It is not possible to add the modifier to the first child, right?